notebook.ipynb

```diff
@@ -0,0 +1,224 @@
+{
```

Comment:
I would call this notebook in a different way, to show what it does (`model_evaluation.ipynb`?)